src/arewefluentyet/milestone.py

```diff
@@ -0,0 +1,74 @@
+import os
+import json
+from datetime import date
+
+
+def parse_date(input):
+    return date(*(int(s) for s in input.split("-")))
+
+
+class Milestone:
+    name = None
+    start_date = None
+    data_path = None
+    progress_data = None
```

Comment:
`name` and `start_date` are good class members, but fold `data_path` and `progress_data` into `__init__`?